src/shared/components/oncoprint/SortUtils.ts

```diff
@@ -302,6 +303,34 @@ export function getClinicalTrackSortComparator(track: ClinicalTrackSpec) {
     };
 }
 
+export function getClinicalTrackInitialSortDirection(
+    track: ClinicalTrackSpec
+): 0 | 1 | -1 {
+    let initialSortOrder: 0 | 1 | -1 = 0;
+    getServerConfig().oncoprint_clinical_tracks_show_by_default.forEach(t => {
+        if (t.stableId === track.attributeId) {
+            if (t.sortOrder === 'ASC') {
+                initialSortOrder = 1;
+            } else if (t.sortOrder === 'DESC') {
+                initialSortOrder = -1;
+            }
+        }
+    });
+    return initialSortOrder;
+}
+
+export function getClinicalTrackInitialShowGapsConfig(
+    track: ClinicalTrackSpec
+): boolean {
+    let gapOn = false;
+    getServerConfig().oncoprint_clinical_tracks_show_by_default.forEach(t => {
+        if (t.stableId === track.attributeId) {
+            gapOn = !!t.gapOn;
+        }
+    });
+    return gapOn;
```

Comment:
```suggestion
    getServerConfig().oncoprint_clinical_tracks_show_by_default.forEach(t => {
        if (t.stableId === track.attributeId) {
            return !!t.gapOn;
        }
    });
    return false;
```
Better to return immediately when the stableId is found.

Reply:
Ah ok.